DTDL/edge/syslog/syslog-stream.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:stream:syslog:stream;1",
+  "@type": "Interface",
+  "displayName": "Syslog Stream",
+  "extends": "dtmi:io:tributech:stream:base;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "SyslogMessageFormat",
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "RFC3164",
```

Comment:
Is this the common name for this format ? Or is there any "customer friendlier" name for the protocol ?

Reply:
I am not aware if there is any user friendly name. Syslog docs use RFCxxxx names. Alternative is to use "old"/"new" format but that is not a way to go

Reply:
Im splitting stream to RFC3164 and RFC5424 streams so I will remove this enum